Libplanet.Action/State/Account.cs

```diff
@@ -15,25 +17,44 @@ namespace Libplanet.Action.State
     /// An internal implementation of <see cref="IAccount"/>.
     /// </summary>
     [Pure]
-    internal class Account : IAccount
+    public class Account : IAccount
     {
-        private readonly IAccountState _baseState;
+        public Account(BlockHash? blockHash, IUnRecordableTrie trie)
+            : this(blockHash, trie, new AccountDelta())
+        {
+        }
+
+        public Account(IAccount baseState)
+            : this(
+                baseState.BlockHash,
+                baseState.Trie,
+                new AccountDelta())
+        {
+        }
 
-        private Account(IAccountState baseState)
-            : this(baseState, new AccountDelta())
+        private Account(IAccount previousAccount, IAccountDelta delta)
+            : this(
+                previousAccount.BlockHash,
+                previousAccount.Trie.Set(delta.ToRawDelta()),
+                delta)
```

Comment:
This seems problematic. Trying to set the whole delta every time would be very bad. 😱

Reply:
I'll leave it as a separate issue.